src/pretix/plugins/sendmail/models.py

```diff
@@ -137,13 +157,15 @@ def send(self):
             status_q |= Q(status=Order.STATUS_PENDING, require_approval=False, valid_if_pending=False,
                           expires__lt=now())
 
+        orders = orders.filter(self.filter_checkins(limit_products))
```

Comment:
See above regarding working on the queryset directly.

```suggestion
        orders = self.filter_checkins(orders)
```